src/themes/mainTheme.ts

```diff
@@ -149,6 +150,13 @@ export const mainTheme = createTheme(
               color: Color.ErrorMain,
             },
           },
+          iconContainer: {
+            opacity: 0.65,
+            '&.Mui-active': {
+              color: Color.PrimaryMain,
```

Comment:
Farge skal være lik uansett om den er aktiv eller ikke(?)
```suggestion
            opacity: 0.65,
            color: Color.PrimaryMain,
            '&.Mui-active': {
```